packages/connectivity/connectivity_macos/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Update package:e2e reference to use the local version in the flutter/plugins
   repository.
+* Remove no-op android folder in the example app.
```

Comment:
Shouldn't this be a new section, not an existing version?

Reply:
0.1.0+5 is never published AFAIK